.luacheckrc

```diff
@@ -11,23 +11,14 @@ exclude_files = {
   "compare-*",
   "sile-*",
```

Comment:
```suggestion
```

This line and the two before it (benchmark and compare) are also artifacts from SILE development and not relevant here